src/i18n/strings/en_EN.json

```diff
@@ -351,6 +351,7 @@
     "To change the permissions in the room, you must be a": "To change the permissions in the room, you must be a",
     "To change the topic, you must be a": "To change the topic, you must be a",
     "To modify widgets in the room, you must be a": "To modify widgets in the room, you must be a",
+    "To remove your own messages, you must be a": "To remove your own messages, you must be a",
```

Comment:
This is technically false. Disabling m.room.redaction disables it for everyone, including yourself.

Reply:
"To remove your own messages, you must be a: [Moderator]". Which bit is false?

(you have to *also* have a separate power level to remove other people's messages, but that's not mentioned here)

Reply:
It's false because the other users setting is affected by the change, but doesn't get mentioned. 

fwiw, I did this exact same change here: https://github.com/matrix-org/matrix-react-sdk/pull/1476/files